src/main/java/com/aws/greengrass/mqttclient/AwsIotMqttClient.java

```diff
@@ -222,7 +243,7 @@ synchronized boolean connected() {
     }
 
     @SuppressWarnings("PMD.NullAssignment")
-    private synchronized void disconnect() {
+    synchronized void disconnect() {
```

Comment:
make it protected if you're wanting to use it in tests.

Reply:
updated likewise